pypeit/edgetrace.py

```diff
@@ -4155,8 +4158,11 @@ def convert_to_tslits_dict(self):
         """
         Stop-gap function to construct the old tslits_dict object.
         """
-        if not self.is_synced:
-            msgs.error('Edges must be synced to construct tslits_dict.')
+
+        # JFH This is causing problems and I cannot figure out how to toggle is_sync when creating psuedo masters. Let's
+        # assume if we are converting to a tslits_dict we are always already synced.
+        #if not self.is_synced:
+        #    msgs.error('Edges must be synced to construct tslits_dict.')
```

Comment:
`is_synced` is a direct check of the object itself; it's not a boolean that can be toggled.  If this is being tripped, it means that the object isn't synced and the output of this method is likely to be wrong in some way.  There are some annoying corner cases where the `sync` method fails and the result is that it's not synced.  If you point me to data where this was causing you problems, I can help you figure out why.  But this check needs to be in place for the tslits_dict conversion to be trustworthy.